backend/middleware/auth_middleware.py

```
@@ -0,0 +1,23 @@
+        if request.url.path.startswith("/auth") or request.url.path.startswith("/docs"):
+            return await call_next(request)
+        
+        token = request.cookies.get("access_token")
+
+        if not token:
+            return JSONResponse(status_code=401, content={"detail": "Not authenticated"})
+
+        try:
+            user_data = verify_jwt_token(token)
+            request.state.user = user_data
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)